test/list-transcription-jobs.test.js

```diff
@@ -0,0 +1,36 @@
+import request from 'supertest';
+import { expect } from 'chai';
+import nock from 'nock';
+import 'dotenv/config';
+
+import { LIST_TRANSCRIBE_RESPONSE } from './utils/helpers';
+
+const args = {
+  Status: 'IN_PROGRESS'
+};
+
+describe('list-transcription-jobs', () => {
+  const { LIST_TRANSCRIBE_URL } = process.env;
+
+  it('should return lists transcription jobs with the specified status', (done) => {
+    nock(LIST_TRANSCRIBE_URL)
```

Comment:
What exactly is being tested in this case?

1. You mock the request response with `nock`
2. You make a request to the `LIST_TRANSCRIBE_URL`
3. What is in the response body of the request? It's the mocked response.

So essentially all the tests here check if the nock library works as expected.